pupil_src/launchables/world.py

```diff
@@ -305,9 +305,12 @@ def handle_notifications(n):
                                         'actor': p.class_name,
                                         'doc': p.on_notify.__doc__})
 
+        width, height = 1280, 720
+        width += icon_bar_width
+        width, height = session_settings.get('window_size', (width, height))
```

Comment:
I get this is cleaner but was the old version not also easy to read? and its more compact.